drivers/mtd_spi_nor/mtd_spi_nor.c

```diff
@@ -303,6 +306,24 @@ static uint32_t mtd_spi_nor_get_size(const mtd_jedec_id_t *id)
         (id->device[1] & ~0x3) == 0) {
         return (0x1F & id->device[0]) * MBIT_AS_BYTES;
     }
+    if (mtd_spi_manuf_match(id, SPI_NOR_JEDEC_MICROCHIP)) {
+        switch (id->device[1]) {
+        case 0x12:  /* SST26VF020A */
+        case 0x8c:  /* SST25VF020B */
+            return  2 * MBIT_AS_BYTES;
```

Comment:
```suggestion
            return 2 * MBIT_AS_BYTES;
```

The CI doesn't like indent after the `return`.